website/sidebars.js

```diff
@@ -122,12 +129,25 @@ export default {
         "how-tos/how-to-calculate-code-hash",
       ],
     },
+    {
+      type: "category",
+      label: "History & Hard Forks",
+      className: "category-history",
+      collapsible: false,
+      items: [
+        "history-and-hard-forks/intro-to-hard-fork",
+        "history-and-hard-forks/ckb-hard-fork-history",
+        "history-and-hard-forks/history-vm-version",
+        "history-and-hard-forks/rethinking-forks",
+      ],
+    },
     {
       type: "category",
       label: "Tech Explanation",
       className: "category-tech-explanation",
       collapsible: false,
       items: [
+        "tech-explanation/fee-estimator",
```

Comment:
Should we move this below "Transaction"?

Reply:
yeah in the long term, but right now this is a new content, so i would like to make it first for a while